WeakAuras/Prototypes.lua

```diff
@@ -5761,7 +5761,7 @@ Private.event_prototypes = {
         trigger.use_cloneId and "true" or "false",
         trigger.use_extend and tonumber(trigger.extend or 0) or 0,
         trigger.use_remaining and "true" or "false",
-        trigger.remaining or 0,
+        type(trigger.remaining) or 0,
```

Comment:
I don't understand the change
`local triggerRemaining = ` will receive "string" or "number" or 0 if it's nil, or am i wrong?
Did you mean to do `type(trigger.remaining) == "number" and trigger.remaining or 0,` or `tonumber(trigger.remaining) or 0` instead?

Reply:
Oh yeah.